File: Externals/4pdaCommon/src/org/softeg/slartus/forpdacommon/NotReportException.java
package org.softeg.slartus.forpdacommon;

import java.io.IOException;

/**
 * User: slinkin
 * Date: 21.10.11
 * Time: 8:03
 */
public class NotReportException extends IOException {
    public NotReportException(String message) {
        super(message);
    }

    public NotReportException(String message, Throwable cause) {
        super(message);
        this.initCause(cause);
    }
}

File: common/common-utils/src/main/java/org/softeg/slartus/forpdacommon/NotReportException.kt
package org.softeg.slartus.forpdacommon

import java.io.IOException

@JvmOverloads
fun notReportError(message: String?, cause: Throwable? = null): Nothing =
    throw NotReportException(message, cause)

open class NotReportException @JvmOverloads constructor(
    message: String?,
    cause: Throwable? = null
) : IOException(message, cause)